### DISCUSSION TITLE
Customize the curvature of smoothstep edges

### DISCUSSION THREAD

**Discussion by piyusht1402** [xyflow/xyflow#3395] [Q&A]
Hi, I wanted to edit the curvature of smooth step edges or basically I wanted to know how can I control the properties of edges for custom edges. There is not any proper example highlighting the use cases where user can modify the property of edges. In the screenshot I have attached, I want to make the edges more rounded or curved on the corners, or if possible define my own custom points of turns for the edges.

Below is the attached image:
![test](https://github.com/wbkd/react-flow/assets/54525525/b9a52b01-470a-4bd1-af02-10ae11561fab)

**Comment by italodeandra** [comment]
I just discovered it.

Add this property to the edge object:

```
pathOptions: {
  borderRadius: 20,
}
```
